Classes/UT3PaladinShield.uc

```diff
@@ -1,9 +1,43 @@
-//============================================================
-// UT3 Paladin Shield
-// Copyright (c) 2012, Jos� Lu�s '100GPing100'
-// Copyright (c) 2014, GreatEmerald
-// Contact: zeluis.100@gmail.com
-//============================================================
+/*
+ * Copyright © 2014 GreatEmerald
+ * Copyright © 2012-2017 Luís 'zeluisping' Guimarães <zeluis.100@gmail.com>

```

Comment:
Thanks for updating the header, but if nothing else changed in the file, your copyright should state only the year 2012.